index.html

```diff
@@ -6052,6 +6100,12 @@ <h3>Executing Script</h3>
  <li><p>If <var>script</var> is not a <a>String</a>,
   return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
+ <li><p>If <var>script</var> is set and <var>name</var> is set,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>If <var>name</var> is set let <var>script</var> be the result
+ of <a>trying</a> to <a>get a known script</a> with argument <var>name</var>
```

Comment:
```suggestion
 of <a>trying</a> to <a>get a script</a> with argument <var>name</var>
```

Reply:
the way I'm looking at it, a pinned script represents a state of a script object not its own object. In which case I would think David's suggestion is the correct fix.